device-types/Digi/Digi-Passport-8.yml

```diff
@@ -0,0 +1,36 @@
+---
+manufacturer: Digi
+model: Digi Passport 8
+slug: digi-passport-8
+part_number: '70002258'
+u_height: 1
+is_full_depth: false
```

Comment:
1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. This appears to be listed in the datasheet (5.19 lbs or 2.28 kg)